src/lib/ptree/ptree.lua

```diff
@@ -366,15 +366,24 @@ function Manager:monitor_worker_counters(id)
    end
 end
 
+local function blacklisted (name)
+   local t = lib.set('macaddr', 'mtu', 'promisc', 'speed', 'status', 'type')
+   name = lib.basename(name)
+   name = name:gsub("%.counter", "")
+   return t[name]
+end
+
 function Manager:sample_active_counters()
    while true do
       local now = rrd.now()
       for name, counters in pairs(self.counters) do
          local sum = counters.archived[0]
-         for pid, active in pairs(counters.active) do
-            local v = counter.read(active)
-            counters.rrd[pid]:add({value=v}, now)
-            sum = sum + v
+         if not blacklisted(name) then
```

Comment:
WDYT about moving the blacklisted check up to Manager:monitor_worker_counters ?  I think in that case we should do nothing for blacklisted counters: no RRD, no aggregated counter, no aggregated RRD, no entry in self.counters.  WDYT?  We'll have to add a check to the `'rm'` case as well of course to do nothing if the counter isn't in self.counters.